web/main/models.py

```diff
@@ -2153,7 +2153,7 @@ def export(self, include_annotations, file_type='docx', export_options=None, is_
         if file_type == 'html':
             return html
 
-        if aws_lambda or settings.FORCE_AWS_LAMBDA_EXPORT:
+        if aws_lambda:
```

Comment:
Okay! To me, this chang means the setting doesn't actually force aws lambda export, it just toggles the default behavior of the view, but if you prefer it this way, fine by me, especially since we are probably ripping out the non lambda code quite soon!

Reply:
Oh, though it -may- affect which version of the export code is run by various tests; let's see.

Reply:
Yeah, it does.
```
INFO     main.models:models.py:3662 Exporting Casebook 1: serializing to HTML
INFO     main.utils:utils.py:665 Exporting Casebook 1: launching pandoc subprocess
```

Reply:
Yeah, this could be de-obfuscated. I can switch this to `DEFAULT_TO_AWS_LAMBDA_EXPORT` (or the like). If we want to force the lambda export, I could add a bit of code to also allow for selecting the template docx.